types/types.go

```diff
@@ -511,6 +511,9 @@ type SystemContext struct {
 	// === dir.Transport overrides ===
 	// DirForceCompress compresses the image layers if set to true
 	DirForceCompress bool
+
+	// Check blobs across registries
+	CheckBlobEverywhere bool
```

Comment:
I prefer calling it `CrossRepositoryBlobCheck`

Reply:
Yes, that's a better name